taxonomies/ingredients.txt

```diff
@@ -9672,15 +9676,6 @@ pt:queijo de pasta dura, queijo duro
 sv:Hårdost
 # fr:fromage-à-pâte-dure has 133 products in 5 languages @2018-10-28
 
-<en:cheese
-fr:fromage râpé
-ca:formatge ratllat
-de:geriebener Käse
-es:queso rallado
-it:formaggio grattugiato
-nl:geraspte kaas
-# fr:fromage-râpé has 185 products in 4 languages @2108-10-28
-
```

Comment:
This kind of pruning still needs to be done for everything that occurs in the processing taxonomy.

Reply:
Yeah, this was just my first attempt at it, because I noticed the english version was already being handled that way.